src/notifications.rs

```diff
@@ -115,11 +116,11 @@ async fn send_notification(
     body: Option<&str>,
     room_id: OwnedRoomId,
     store: &AsyncProgramStore,
-    play_sound: bool,
+    sound_hint: String,
```

Comment:
```suggestion
    sound_hint: Option<&str>,
```
Using only a reference instead of the owned data is only a nitpick since it doesn't make a difference. But in my opinion, functions should only take owned data if they need it since it might reduce cloning if the caller has no owned version.